### PR TITLE
Update main.cpp to include Italian

### PR DESCRIPTION
**Pull request by KintsugiUwU**
Added Italian translation, thanks Spanish guys

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -105,6 +105,8 @@ static const char * get_nintendo_network_message() {
             return "Usando Nintendo Network";
         case nn::swkbd::LanguageType::French:
             return "Sur Nintendo Network";
+         case nn::swkbd::LanguageType::Italian:
+            return "Usando Nintendo Network";
```
Comment (KintsugiUwU) [inline src/main.cpp:109]: Could also use "Usando il Nintendo Network", which translates to "Using the Nintendo Network", maybe a more correct form

File: src/main.cpp [hunk 2]
```diff
@@ -117,6 +119,8 @@ static const char * get_pretendo_message() {
             return "Usando Pretendo Network";
         case nn::swkbd::LanguageType::French:
             return "Sur Pretendo Network";
+        case nn::swkbd::LanguageType::Italian:
+            return "Usando Pretendo Network";
```
Comment (KintsugiUwU) [inline src/main.cpp:123]: Could also use "Usando il Pretendo Network", which translates to "Using the Pretendo Network", maybe a more correct form

**Comment by jonbarrow** [comment]
Calling in @gitlimes to verify this as our native Italian

**Comment by gitlimes** [comment]
Works for me, although it sounds a bit unnatural...

May I suggest "Connesso a Nintendo Network" (Connected to Nintendo Network) as an alternative?

**Comment by KintsugiUwU** [comment]
I just wen by he English version

Using Nintendo Network
Using Pretendo Network

Using is "usare", if we want to use "connesso al Nintendo Network" the English version should be connected to ....

**Comment by gitlimes** [comment]
Localizations don't have to be literal translations. They should convey the same meaning, while sounding natural in the target language.

With that said, your original version might actually be better than my proposal, as telling the user they're "connected" to the service without actually checking for a connection could be misleading.

TL;DR: LGTM!

**Comment by KintsugiUwU** [comment]
You're right, localizations don't have to be 1:1 translations.

**Comment by ashquarky** [comment]
I agree that "Connected" is probably not the best language to use, since we don't actually know that.

This looks good then!
